app/views/steps/application/without_notice_details/edit.html.erb

```diff
@@ -1,33 +1,28 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
 
-    <%= step_form @form_object do |f| %>
-      <%= f.text_area :without_notice_details, size: '40x4', class: 'form-control-3-4' %>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_text_area :without_notice_details %>
 
-      <%=
-        f.radio_button_fieldset :without_notice_frustrate, inline: true do |fieldset|
-          fieldset.radio_input(GenericYesNo::YES, panel_id: :without_notice_frustrate_panel)
-          fieldset.radio_input(GenericYesNo::NO)
-          fieldset.revealing_panel(:without_notice_frustrate_panel) do |panel|
-            panel.text_area :without_notice_frustrate_details, size: '40x4', class: 'form-control-3-4'
-          end
-        end
-      %>
+      <%= f.govuk_radio_buttons_fieldset :without_notice_frustrate, legend: { tag: 'span', size: 'm' } do %>
+        <%= f.govuk_radio_button :without_notice_frustrate, GenericYesNo::YES, link_errors: true %>
```

Comment:
I think this is the same issue as per my comment here:
https://github.com/ministryofjustice/c100-application/pull/900#discussion_r418512802

The text area needs to open with the YES radio option.